2013/02/24 16:45:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:45:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:45:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:45:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:45:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:45:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:49:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:49:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:49:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:50:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:50:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:51:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:51:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:51:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:51:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:53:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:53:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:54:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:54:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:54:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:54:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:54:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:57:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:57:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:57:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:57:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:57:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:57:46 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php:337
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(310): CController->invalidActionParams(Object(CInlineAction))
#1 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilter.php(41): CFilterChain->run()
#3 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#4 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#5 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(292): CFilterChain->run()
#7 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(283): CController->run('edit')
#9 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('cms/fields/edit')
#10 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/admin/fields/edit
HTTP_REFERER=http://portfolio.local/admin/fields/edit/22
---
2013/02/24 16:57:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:58:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:58:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:58:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:58:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:58:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:58:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:58:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/24 16:58:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:38:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:38:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:38:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:38:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:39:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:39:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:39:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:39:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:42:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:42:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:42:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:42:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:42:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 06:53:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:09:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:09:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:09:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:19:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:19:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "posts/getmeta".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('posts/getmeta')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/posts/getmeta?cat=[object%20Object]
HTTP_REFERER=http://portfolio.local/admin/posts/edit/10
---
2013/02/25 07:19:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:19:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "posts/getmeta".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('posts/getmeta')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/posts/getmeta?cat=[object%20Object]
HTTP_REFERER=http://portfolio.local/admin/posts/edit/10
---
2013/02/25 07:20:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:20:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "posts/getmeta".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('posts/getmeta')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/posts/getmeta?cat=6,7
HTTP_REFERER=http://portfolio.local/admin/posts/edit/10
---
2013/02/25 07:20:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:20:45 [error] [php] Invalid argument supplied for foreach() (D:\xampp\htdocs\portfolio-web-apps\apps\protected\modules\cms\controllers\PostsController.php:59)
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(134): PostsController->runAction()
#1 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilter.php(41): CFilterChain->run()
#2 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(1146): CAccessControlFilter->filter()
#3 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CInlineFilter.php(59): PostsController->filterAccessControl()
#4 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#5 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(266): PostsController->runActionWithFilters()
#7 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(283): PostsController->run()
#8 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController()
#9 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CWebApplication->run()
REQUEST_URI=/admin/posts/getmeta?cat=6,7
in D:\xampp\htdocs\portfolio-web-apps\apps\protected\modules\cms\controllers\PostsController.php (59)
in D:\xampp\htdocs\portfolio-web-apps\apps\index.php (13)
2013/02/25 07:21:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:21:23 [error] [php] Use of undefined constant conditions - assumed 'conditions' (D:\xampp\htdocs\portfolio-web-apps\apps\protected\modules\cms\controllers\PostsController.php:80)
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(134): PostsController->runAction()
#1 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilter.php(41): CFilterChain->run()
#2 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(1146): CAccessControlFilter->filter()
#3 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CInlineFilter.php(59): PostsController->filterAccessControl()
#4 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#5 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(266): PostsController->runActionWithFilters()
#7 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(283): PostsController->run()
#8 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController()
#9 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CWebApplication->run()
REQUEST_URI=/admin/posts/getmeta?cat=6,7
in D:\xampp\htdocs\portfolio-web-apps\apps\protected\modules\cms\controllers\PostsController.php (80)
in D:\xampp\htdocs\portfolio-web-apps\apps\index.php (13)
2013/02/25 07:21:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:21:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:21:48 [error] [php] Use of undefined constant conditions - assumed 'conditions' (D:\xampp\htdocs\portfolio-web-apps\apps\protected\modules\cms\controllers\PostsController.php:80)
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(134): PostsController->runAction()
#1 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilter.php(41): CFilterChain->run()
#2 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(1146): CAccessControlFilter->filter()
#3 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CInlineFilter.php(59): PostsController->filterAccessControl()
#4 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#5 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(266): PostsController->runActionWithFilters()
#7 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(283): PostsController->run()
#8 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController()
#9 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CWebApplication->run()
REQUEST_URI=/admin/posts/getmeta?cat=6,7
in D:\xampp\htdocs\portfolio-web-apps\apps\protected\modules\cms\controllers\PostsController.php (80)
in D:\xampp\htdocs\portfolio-web-apps\apps\index.php (13)
2013/02/25 07:22:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:22:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:22:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:22:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:23:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:23:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:24:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:24:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:24:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:25:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:25:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:25:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:26:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:26:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:26:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:27:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:30:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:30:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:30:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:30:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:30:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 07:31:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 10:11:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 10:20:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 10:20:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 10:33:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 10:34:02 [error] [php] Trying to get property of non-object (D:\xampp\htdocs\portfolio-web-apps\apps\protected\modules\cms\controllers\PostsController.php:99)
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(134): PostsController->runAction()
#1 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilter.php(41): CFilterChain->run()
#2 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(1146): CAccessControlFilter->filter()
#3 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CInlineFilter.php(59): PostsController->filterAccessControl()
#4 D:\xampp\htdocs\portfolio-web-apps\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#5 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\htdocs\portfolio-web-apps\framework\web\CController.php(266): PostsController->runActionWithFilters()
#7 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(283): PostsController->run()
#8 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController()
#9 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CWebApplication->run()
REQUEST_URI=/admin/posts/getmeta?cat=6,7&post_id=10
in D:\xampp\htdocs\portfolio-web-apps\apps\protected\modules\cms\controllers\PostsController.php (99)
in D:\xampp\htdocs\portfolio-web-apps\apps\index.php (13)
2013/02/25 10:34:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 10:34:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 10:34:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 10:34:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 10:34:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:50:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:50:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:51:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:52:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:52:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:52:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:52:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:52:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:52:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:52:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:53:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:53:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:53:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:53:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:55:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:55:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:56:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:58:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:58:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:58:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:59:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:59:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:59:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:59:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:59:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:59:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/02/25 11:59:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\xampp\htdocs\portfolio-web-apps\framework\web\CWebApplication.php(142): CWebApplication->runController('favicon.ico')
#1 D:\xampp\htdocs\portfolio-web-apps\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\portfolio-web-apps\apps\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
